bindings/rust/s2n-tls/src/connection.rs

```diff
@@ -875,6 +875,13 @@ impl Connection {
             .into_result()?;
         Ok(self)
     }
+
+    pub fn quic_process_post_handshake_message(&mut self) -> Result<&mut Self, Error> {
```

Comment:
Note: Why am I adding this function to our higher-level bindings? Because this function takes in a s2n_connection, which is a private field on Connection. So I'd either have to make that field public or provide a getter method to access it, both of which feel more dangerous than adding this function to the higher-level bindings. It is still gated with the "quic" feature.